public/js/app.js

```diff
@@ -0,0 +1,315 @@
+                if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+                    document.getElementById("toTop").style.display = "block";
+                } else {
+                    document.getElementById("toTop").style.display = "none";
+                }
+            }
+
+            /*************************HEADER**********************/
+
+            $('#header').hide();
+            var hauteur = 580;
+            $(function () {
+                $(window).scroll(function () {
+                    if ($(this).scrollTop() > hauteur) {
+                        $('#header').css("display", "flex")
+                         .fadeIn();
+                    } else {
+                        $('#header').hide();
+                    }
+                });
+            });
+
+            /********** Scroll smooth sections ************/
+
+            $(document).on('click', 'a[href^="#"]:not(li.nav-item a)', function (e) {
+                var id = $(this).attr('href');
+
+                var $id = $(id);
+                if ($id.length === 0) {
+                    return;
+                }
+
+                e.preventDefault();
+
+                var pos = $id.offset().top;
+
+                $('body, html').animate({
+                    scrollTop: pos
+                }, 1000);
+                
+                if ($('#hamburger-overlay').length) {
+                    $('#hamburger').removeClass('hamburger-activated');
+                }
+
+            });
+
+
+            /**************** Competences items ***************/
+
+            $(window).scroll(function () {
+                var wPos = $(window).scrollTop();
+                var rdPos = $('.compItems').offset().top;
+
+                if (wPos + 500 >= rdPos) {
+                    var art = $('.compItems').find('li');
+                    art.addClass('inView');
+                }
+                
+            });
+
+            /********** Formulaire contact ************/
+            $('input').blur(function () {
+                if ($(this).val() == '') {
+                    $(this).addClass('wrong');
+                } else {
+                    $(this).removeClass('worng');
+                }
+            });
+
+            $("#contactForm").submit(function (e) {
+                e.preventDefault();
+                var nom = $("#nom").val();
+                var email = $("#email").val();
+                var message = $("#message").val();
+                var dataString = nom + email + message;
+                var msg;
+
+                if (dataString == "") {
+                    msg = $('<div class="echec">Vous devez remplir tous les champs</div>');
+
+                } else if (nom == "") {
+                    msg = $('<div class="echec">Vous devez saisir vos nom et prénom</div>');
+                } else if (email == "") {
+                    msg = $('<div class="echec">Vous devez saisir votre mail</div>');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/vanmarcke/ocr_projet5_blog/pullRequest?prid=8116981)
```suggestion
                    msg = $("<div class=\"echec\">Vous devez saisir votre mail</div>");
```